README.md

```diff
@@ -452,6 +472,39 @@ export default MyComponent = () => {
 
 ## Migrating between versions
 
+### Migrating from 4.x to 5.x
+
```

Comment:
Since there are three specific changes that just about everyone will need to make, I'd call them out here, then have your specific breakdowns below.

Something like:

```
There are three notable changes required to your application code when migrating from okta-react 4.x to okta-react 5.x:
- pass a `navigate` prop to the `Security` component
- import `SecureRoute` from the new location (SecureRoute is specific for using react-router)
- pass a `useOktaAuth` prop to the `SecureRoute` Component
```
Then have each section below as a level 4 subheading.